src/main/java/org/sonarqube/A.java

```diff
@@ -3,7 +3,11 @@
 public class A {
   String message = "foo";
 
+  // TODO fix this
   public String foo() {
+    message = null;
+    message.length();
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: The return value of "length" must be used. ([squid:S2201](http://localhost:9000/coding_rules#rule_key=squid%3AS2201))

[See it in SonarQube](http://localhost:9000/project/issues?id=org.sonarqube%3Apr-decoration&branch=pr_29&issues=AWBUGpoFvB3VRKxIB7gQ&open=AWBUGpoFvB3VRKxIB7gQ)

[ISSUE_KEY]: AWBUGpoFvB3VRKxIB7gQ

Reply:
===RESOLVED===
This issue has been resolved by a user in SonarQube